# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/sjog/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/sjog_brightness.c.orig	2004-09-11 01:34:17.916580888 +0200
+++ ./src/sjog_brightness.c	2004-09-11 01:34:38.650428864 +0200
@@ -52,7 +52,7 @@
         break;
 
      default:
-
+	;
    }
 
 }
--- ./src/sjog_scroll.c.orig	2004-09-11 01:37:04.256293408 +0200
+++ ./src/sjog_scroll.c	2004-09-11 01:37:16.980359056 +0200
@@ -64,7 +64,7 @@
         break;
 
      default:
-
+	;
    }
 
 }
